chrome.ts

```diff
@@ -238,7 +238,7 @@ class ChromeImpl implements Chrome {
         return;
       }
 
-      const folder = entry.folder;
+      const folder = entry.folder?.split("/").slice(1).join();
```

Comment:
In any case, I think this bug should be fixed in a separate PR. :thinking:  So, could you please revert this change? (I'll open an issue later about this bug)
```suggestion
      const folder;
```